eng/scripts/Collect-Changelogs.ps1

```diff
@@ -0,0 +1,41 @@
+param($month)
+
+$repoRoot = Resolve-Path "$PSScriptRoot/../..";
+. ${repoRoot}\eng\common\scripts\SemVer.ps1
+. ${repoRoot}\eng\common\scripts\ChangeLog-Operations.ps1
+$InstallNotes = "";
+$ReleaseNotes = "";
+
+$date = Get-Date -Month $month -Format "yyyy-MM"
+$date += "-\d\d"
+
+Get-ChildItem "$repoRoot/sdk" -Filter CHANGELOG.md -Recurse | Sort-Object -Property Name | % {
+    
+    $changeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $_ 
+    $package = $_.Directory.Name
+    $serviceDirectory = $_.Directory.Parent.Name
+
+    foreach ($changeLogEntry in $changeLogEntries.Values)
+    {
+        if ($changeLogEntry.ReleaseStatus -notmatch $date)
+        {
+            
+            continue;
+        }
+
+        $version = $changeLogEntry.ReleaseVersion
+        $githubAnchor = $changeLogEntry.ReleaseTitle.Replace("## ", "").Replace(".", "").Replace("(", "").Replace(")", "").Replace(" ", "-")
+
+        $InstallNotes += "$> dotnet add package $package --version $version`n";
+        $ReleaseNotes += "### $package [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/$serviceDirectory/$package/CHANGELOG.md#$githubAnchor)`n"
+        $changeLogEntry.ReleaseContent | %{ 
+
+            $ReleaseNotes += $_.Replace("###", "####")
+            $ReleaseNotes += "`n"            
+        }
+        $ReleaseNotes += "`n"
+    }
+}
+
+$InstallNotes | Write-Host;
```

Comment:
Tip: you don't actually have to pipe to `write-host` - that forces it to write to stdout. You can just return it, in which case the output writes to the host automatically, but you can also then use the strings in other scripts if you wanted, like so:

```powershell
$installNotes, $releaseNotes = eng/scripts/Collect-ChangeLogs.ps1 9
```

In fact, rarely should use write-host. write-verbose is usually better for automation, but if you really want to make sure it's always visible, write-host is a good replacement for write-verbose.

Reply:
Nice! Great idea.

Reply:
Out of curious @heaths do you know the magic setting to make Write-Verbose log? I seem to never be able to enable it for an entire script run.